notifier/selfstate/selfstate.go

```diff
@@ -24,19 +28,25 @@ type SelfCheckWorker struct {
 	Notifier   notifier.Notifier
 	Config     Config
 	tomb       tomb.Tomb
-	Heartbeats []heartbeat.Heartbeater
+	heartbeats []heartbeat.Heartbeater
+}
+
+func NewSelfCheckWorker(logger moira.Logger, database moira.Database, notifier notifier.Notifier, config Config) *SelfCheckWorker {
+	heartbeats := createStandardHeartbeats(logger, database, config)
+	return &SelfCheckWorker{Logger: logger, Database: database, Notifier: notifier, Config: config, heartbeats: heartbeats}
 }
 
 // Start self check worker
 func (selfCheck *SelfCheckWorker) Start() error {
 	if !selfCheck.Config.Enabled {
-		selfCheck.Logger.Debugf("Moira Self State Monitoring disabled")
-		return nil
+		selfCheck.Logger.Debugf(ErrDisabled.Error())
+		return ErrDisabled
 	}
+
 	senders := selfCheck.Notifier.GetSenders()
 	if err := selfCheck.Config.checkConfig(senders); err != nil {
-		selfCheck.Logger.Errorf("Can't configure Moira Self State Monitoring: %s", err.Error())
-		return nil
+		selfCheck.Logger.Errorf(ErrWrongConfig.Error())
```

Comment:
Will be logged twice?

Reply:
yes, thanks. Fixed here https://github.com/moira-alert/moira/pull/774/commits/e44f41735e609b58dd48bb95a35008b101f7af5c